lib/tools/ui/widgets/image_picker_on_tap.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:myecl/tools/constants.dart';
+import 'package:myecl/tools/functions.dart';
+
+class ImagePickerOnTap extends StatelessWidget {
+  final ImagePicker picker;
+  final ValueNotifier<Uint8List?> imageBytesNotifier;
+  final ValueNotifier<Image?> imageNotifier;
+  final void Function(TypeMsg, String) displayToastWithContext;
```

Comment:
You are not using it at the moment.
I think you can just initialize it in the Widget and not asking it to be passed as an argument

Reply:
Good catch, I forgot to use it.
I don't see how to initialize it in the widget as I do not have access to the context which is required for the function `displayToast`.

Reply:
You are right ! My bad